packages/cli/generators/controller/templates/src/controllers/controller-rest-template.ts.ejs

```diff
@@ -68,10 +68,10 @@ export class <%= className %>Controller {
     },
   })
   async updateAll(
-    @requestBody() <%= name %>: <%= modelName %>,
+    @requestBody() data: <%= modelName %>,
```

Comment:
Not a big deal but I personally prefer not using a generic name like `data` since it would repeat multiple times and as such would much rather have it be the model name ... `todo: Todo`.

Reply:
Makes sense, will go with `todo: Todo`.

Before my change, we were using the controller name as the argument name, e.g. `TodoController: Todo`.